administration/src/cards/CardBlueprint.ts

```diff
@@ -118,32 +115,15 @@ export class CardBlueprint {
       // Extensions valid
       this.extensions.every(ext => ext.isValid()) &&
       // Expiration date valid
-      (this.isExpirationDateValid() || this.hasInfiniteLifetime()) &&
-      // Number of bytes is valid
-      this.hasValidSize()
+      (this.isExpirationDateValid() || this.hasInfiniteLifetime())
     )
   }
 
-  hasValidSize(): boolean {
-    // See https://github.com/digitalfabrik/entitlementcard/issues/690  for more context.
-
-    const dynamicCode = new QrCode({
-      qrCode: { value: this.generateActivationCode(), case: 'dynamicActivationCode' },
-    }).toBinary()
-
-    const staticCode = new QrCode({
-      qrCode: { value: this.generateStaticVerificationCode(), case: 'staticVerificationCode' },
-    }).toBinary()
-
-    return isContentLengthValid(dynamicCode) && isContentLengthValid(staticCode)
-  }
-
   generateCardInfo = (): CardInfo => {
     const extensionsMessage: PartialMessage<CardExtensions> = {}
 
     this.extensions.forEach(extension => {
-      if (extension.state === null || !extension.setProtobufData) {
-        // We allow to skip invalid extensions to enable computing the protobuf size.
+      if (!extension.setProtobufData) {
```

Comment:
It should not be removed (to allow a partial correct state). Although, now there are no errors because of removing this, as the protobuf computation has been removed.